src/app/games/trexo/TrexoTutorial.ts

```diff
@@ -0,0 +1,95 @@
+            TrexoState.getInitialState(),
+        ),
+        TutorialStep.anyMove(
+            $localize`Dropping a tile`,
+            $localize`When you drop a tile, it needs to be on even ground, and it cannot be right on top of another tile. In others words, it needs to be either on the floor, or on two tiles at the same height. To do that, just click on the square where you want to put the opponent side of the tile, then on the neighboring square where you want to put your piece.<br/><br/>You're playing Dark, put a tile on the board.`,
+            TrexoState.getInitialState(),
+            TrexoMove.from(new Coord(4, 4), new Coord(3, 4)).get(),
+            $localize`Congratulations!`,
+        ),
+        TutorialStep.fromMove(
+            $localize`Dropping a piece over other pieces`,
+            $localize`You can put a tile on other tiles. For that you must respect two rules:<br/><ul><li>The two pieces must be at the same level.</li><li>You cannot drop your piece on only one other piece.</li></ul><br/>You're playing Dark, do such a move!`,
+            new TrexoState([
+                [______, ______, ______, ONE__0, ZERO_0, ZERO_1, ONE__1, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+                [______, ______, ______, ______, ______, ______, ______, ______, ______, ______],
+            ], 7),
+            [
+                TrexoMove.from(new Coord(4, 0), new Coord(5, 0)).get(),
+                TrexoMove.from(new Coord(5, 0), new Coord(4, 0)).get(),
+            ],
+            $localize`Congratulations!`,
+            $localize`Incorrect, you put that piece on the floor level, please put it on top of two pieces.`,
+        ),
+        TutorialStep.fromPredicate(
+            $localize`Victory`,
+            $localize`To win at Trexo, you need to align 5 of your pieces. Importantly, only the piece on top counts, the hidden piece are no longer threats, and a victory can include pieces on differents heights. Since you have to put the piece of your opponent first, if you create a victory for them and for you at the same turn, they'll be considered the winners. If you create only a victory for them too, obviously.<br/><br/>You're playing Light, win.`,
```

Comment:
hidden pieces*

Reply:
et tu trouve pas que le « If you create only a victory for them too, obviously » est un peu out of place ?
Dans la construction de phrase on pourrait croire que c'est lié au fait  qu'on joue la pièce adverse en premier, alors que ça n'a rien à voir, c'est simplement qu'on a fait un alignement adverse

Reply:
Il est out of place, j'ai amélioré ça selon ta suggestion et c'est bien mieux!